examples/event_informer.rs

```diff
@@ -14,13 +17,12 @@ async fn main() -> anyhow::Result<()> {
     let client = APIClient::new(config);
 
     let events = Api::v1Event(client);
-    let ei = Informer::new(events)
-        .init().await?;
+    let ei = Informer::new(events).init().await?;
 
     loop {
-        ei.poll().await?;
+        let mut events = ei.poll().await?.boxed();
 
-        while let Some(event) = ei.pop() {
+        while let Some(Ok(event)) = events.next().await {
```

Comment:
maybe we should slap a question mark on here, and have the poll restarted if we get error items

Reply:
👍